x-pack/plugins/apm/ftr_e2e/cypress/integration/service_overview/time_comparison.spec.ts

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import url from 'url';
+import archives_metadata from '../../fixtures/es_archiver/archives_metadata';
+const { start, end } = archives_metadata['apm_8.0.0'];
+
+const baseUrl = url.format({
+  pathname: '/app/apm/services/opbeans-java/overview',
+  query: { rangeFrom: start, rangeTo: end },
+});
+
+const apisToIntercept = [
+  {
+    endpoint: '/api/apm/services/opbeans-java/transactions/charts/latency',
+    as: 'latencyChartRequest',
+  },
+  {
+    endpoint: '/api/apm/services/opbeans-java/throughput',
+    as: 'throughputChartRequest',
+  },
+  {
+    endpoint: '/api/apm/services/opbeans-java/transactions/charts/error_rate',
+    as: 'errorRateChartRequest',
+  },
+  {
+    endpoint:
+      '/api/apm/services/opbeans-java/transactions/groups/detailed_statistics',
+    as: 'transactionGroupsDetailedRequest',
+  },
+  {
+    endpoint: '/api/apm/services/opbeans-java/error_groups/detailed_statistics',
+    as: 'errorGroupsDetailedRequest',
+  },
+  {
+    endpoint:
+      '/api/apm/services/opbeans-java/service_overview_instances/detailed_statistics',
+    as: 'instancesDetailedRequest',
+  },
+];
+
+describe('Service overview: Time Comparison', () => {
+  it('enables by default the time comparison feature with Last 24 hours selected', () => {
+    cy.visit(baseUrl);
+    cy.url().should('include', 'comparisonEnabled=true&comparisonType=day');
+  });
+
+  it('toggles comparison off disables select box and APIs are called without comparison time range', () => {
```

Comment:
I prefer to break these up if when I read it it doesn't make sense as a sentence, maybe something like:

```js
describe('when comparison is toggled off', () => {
  it('disables the comparison select and calls the APIs without a comparison time range', () => {
```

The "and" in there makes me want to break it up into two separate `it`s but I'm more tolerant of this in slow e2e tests.